tests/e2e/java_test.go

```diff
@@ -50,8 +56,43 @@ var _ = Describe("odoJavaE2e", func() {
 			runCmd("odo delete javaee-git-test -f")
 		})
 
+		It("Should be able to deploy a git repo that contains a wildfly application without wait flag", func() {
+
+			// Deploy the git repo / wildfly example
+			cmpCreateLog := runCmd("odo create wildfly wo-wait-javaee-git-test --git " + warGitRepo)
+			Expect(cmpCreateLog).Should(ContainSubstring("This may take few moments to be ready"))
+			buildName := runCmd("oc get builds --output='name' | grep wo-wait-javaee-git-test | cut -d \"/\" -f 2")
+			Expect(buildName).To(ContainSubstring("wo-wait-javaee-git-test"))
```

Comment:
Also please check that the build status is running

Reply:
@syamgk Why was this marked as resolved?

Reply:
i've added the status check in line number 71

Reply:
okey I
 got it the 'oc get build' and build status

Reply:
:+1: